pyfolio/deprecate.py

```diff
@@ -0,0 +1,45 @@
+"""Utilities for marking deprecated functions."""
+# Copyright 2016 Quantopian, Inc.
```

Comment:
I think we can kill this copyright comment here actually, from what I understand we only need to have this once in the README. @richafrank @twiecki any info on this?